.github/workflows/php-unit-tests.yml

```diff
@@ -35,24 +35,42 @@ jobs:
         with:
           slug: wordpress
           
+  GetWCMatrix:
+    name: Get WC L-1 version Matrix
+    runs-on: ubuntu-latest
+    outputs:
+      previous-wc-version: ${{ fromJson(steps.wc.outputs.versions)[1] }}
+      latest-wc-version: ${{ fromJson(steps.wc.outputs.versions)[0] }}
+    steps:
+      - name: Get Release versions from WooCommerce
+        id: wc
+        uses: woocommerce/grow/get-plugin-releases@actions-v1
+        with:
+          slug: woocommerce 
+          releases: 2
+          
   UnitTests:
-    name: PHP unit tests - PHP ${{ matrix.php }}, WP ${{ matrix.wp-version }}
-    needs: GetMatrix
+    name: PHP unit tests - PHP ${{ matrix.php }}, WP ${{ matrix.wp-version }}, WC ${{ matrix.wc-version || 'latest' }}
+    needs: [GetWCMatrix, GetWPMatrix]
     runs-on: ubuntu-latest
     env:
       WP_CORE_DIR: "/tmp/wordpress/src"
       WP_TESTS_DIR: "/tmp/wordpress/tests/phpunit"
     strategy:
       matrix:
         php: [8.0]
-        wp-version: ${{ fromJson(needs.GetMatrix.outputs.wp-versions) }}
+        wp-version: ${{ fromJson(needs.GetWPMatrix.outputs.wp-versions) }}
+        wc-version: [ latest ]
         include:
           - php: 7.4
-            wp-version: ${{ needs.GetMatrix.outputs.latest-wp-version }}
+            wp-version: ${{ needs.GetWPMatrix.outputs.latest-wp-version }}
+            wc-version: ${{ needs.GetWCMatrix.outputs.previous-wc-version }}
```

Comment:
In the interest of not testing all combinations (to save billing minutes), on other extensions we make this test do the oldest supported version of everything. So in that case wouldn't this need to be PHP 7.4, WP L-2, WC L-1.
Since this is the only test that uses an older version of WC I wouldn't be against changing this to WC L-2.

Reply:
I changed the matrix, now we follow the same as Automatewoo